core/app/controllers/spree/base_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require 'cancan'
+require 'jwt'
```

Comment:
why this is needed here?

Reply:
Probably not, I wondered that myself

Reply:
Do `p require 'jwt'`. If you get `false`, you'll know soon enough.